detection_rules/devtools.py

```diff
@@ -820,17 +822,39 @@ def raw_permalink(raw_link):
         link_name = name.split('.')[0]
         markdown_links.append(f'|[{link_name}]({url})|')
 
+    markdown = [
+        f'**Full coverage**: {NAVIGATOR_BADGE}',
+        '\n',
+        f'**Coverage by platform**: [navigator]({platforms_url})',
+        '\n',
+        '| other navigator links by rule attributes |',
+        '|------------------------------------------|',
+    ] + markdown_links
+
     if print_markdown:
-        markdown = [
-            f'**Full coverage**: {NAVIGATOR_BADGE}',
-            '\n',
-            f'**Coverage by platform**: [navigator]({platforms_url})',
-            '\n',
-            '| other navigator links by rule attributes |',
-            '|------------------------------------------|',
-        ] + markdown_links
         click.echo('\n'.join(markdown) + '\n')
 
+    if update_coverage:
+        coverage_file_path = get_path('docs', 'ATT&CK-coverage.md')
+        header_lines = """# Rule coverage
+
+ATT&CK navigator layer files are generated when a package is built with `make release` or `python -m detection-rules`.
+This also means they can be downloaded from all successful builds.
+
+These files can be used to pass to a custom navigator
+session. For convenience, the links are generated below. You can also include multiple across tabs in a single session,
+though it is not advisable to upload _all_ of them as it will likely overload your browsers resources.
+
+## Current rule coverage
+
+The source files for these links are regenerated with every successful merge to main. These represent coverage from the
+state of rules in the `main` branch."""
```

Comment:
to keep the formatting nice, we should use [`textwrap.dedent()`](https://docs.python.org/3/library/textwrap.html#textwrap.dedent) here:

```python
    if update_coverage:
        # ...
        header_lines = textwrap.dedent("""
            # Rule coverage

            ATT&CK navigator layer files are generated when a package is built with
            `make release` or `python -m detection-rules`. This also means they can
            be downloaded from all successful builds. These files can be used to
            pass to a custom navigator session. For convenience, the links are
            generated below. You can also include multiple across tabs in a single
            session, though it is not advisable to upload _all_ of them as it will
            likely overload your browsers resources.

            ...
        """)
```

`textwrap.dedent()` will remove shared indent from all text lines, so the text will not have any margin on the left